apps/hls_examples/blend_hls/run_zynq.cpp

```diff
@@ -0,0 +1,68 @@
+#include <cstdio>
+#include <cstdlib>
+#include <cassert>
+#include <math.h>
+
+#include <fcntl.h>
+#include <unistd.h>
+#include "pipeline_zynq_seedark.h"
+#include "seedark_pipeline.h"
+
+#include "HalideBuffer.h"
+#include "halide_image_io.h"
+
+using namespace Halide::Tools;
+using namespace Halide::Runtime;
+
+extern "C" {
+extern int halide_zynq_init();
+}
+
+using namespace Halide::Tools;
+
+int main(int argc, char **argv) {
+    // Open the buffer allocation device
+    halide_zynq_init();
+
+    Buffer<uint8_t> input1 = load_image(argv[1]);
+    Buffer<uint8_t> input2 = load_image(argv[2]);
+    Buffer<uint8_t> weight(2, 1);
+    weight(0) = 5;
+    weight(1) = 95;
+    
+    Buffer<uint8_t> out_native(input1.width(), input1.height(), 3);
+    Buffer<uint8_t> out_zynq(input1.width(), input1.height(), 3);
+    printf("start.\n");
+
+    blend_cpu(input1, input2, weight, out_native);
+    save_image(out_native, "out.png");
+    printf("cpu program results saved.\n");
+    //out_native = load_image("out_native.png");
```

Comment:
Delete? We should in general avoid check in code that is commented.

Reply:
Good catch. I will remove that.